packages/components/app/styles/components/pagination.scss

```diff
@@ -70,10 +70,18 @@
     color: var(--token-color-foreground-action-active);
   }
 
+  // IMPORTANT: we need to use also the [disabled] selector because if the "disabled" attribute is applied to the control that,
+  // since internally it's using the `Hds::Interactive` component, it can be a link or a button, depending on the parameters provided, but
+  // the ":disabled" pseudo-selector is not applied to the element in the browser (rightly) because a link can't be disabled
+  // but from the product perspective there may be use cases where they need to have a the control as a link that looks disabled anyway
+
   &:disabled,
+  &[disabled],
   &.mock-disabled {
     color: var(--token-color-foreground-disabled);
     cursor: not-allowed;
+    // TODO discuss with Kristin if we want to do it (removes the cursor too!)
+    pointer-events: none;
   }
```

Comment:
I added a comment to the PR with my test code as an example. I would prefer a solution which doesn't add an invalid attribute (the `disabled` attribute) to anchor link tags.

(Cloud-ui's Pagination also renders a button tag with the `disabled` attribute when the Prev & Next controls are disabled.)

Reply:
Not sure if it helps with the matter, but in my experience `<a role="link" aria-disabled="true">Link</a>` is one of the few solutions that convey a disabled link to screen reader users.

Reply:
Good to know. Would you need to add the role though since it's already a link? Or is that needed so that the screen reader both announces that it's a link AND that it's disabled.

It would still require extra work though as the link isn't truly disabled. (If a user clicks on it, a valid link would still work.)

Reply:
If the anchor doesn't have an href it is often treated as an inline element (similar to a span) which is why we're adding the role. The whole example is not needed anymore though, as we seem to be always rendering buttons for disabled states.

Reply:
This (the `pointer-events: none` and the disabled link) is outdated now, we are showing a disabled button. Resolving the conversation.